src/Components/Facility/ConsultationDetails.tsx

```diff
@@ -901,9 +901,9 @@ export const ConsultationDetails = (props: any) => {
             )}
             {consultationData.discharge_advice && (
               <div className="mt-4">
-                <h3 className="flex text-lg font-semibold leading-relaxed text-gray-900">
+                <h3 className="flex flex-wrap justify-between text-lg font-semibold leading-relaxed text-gray-900">
```

Comment:
Instead of applying display flex to the `<h3>`.
Structure like
```jsx
<div className="flex">
   <h3>Prescription</h3>
   <h3>
         <i className="fas fa-history text-sm pr-2"></i>
          {consultationData.modified_date && moment(consultationData.modified_date).format("lll")}
   </h3>
</div>
```